### PR TITLE
disable PR comments

### PR DESCRIPTION
**Pull request by mhassaninmsft**

### CODE REVIEW

File: .github/workflows/ci_frontend.yml [hunk 1]
```diff
@@ -1,16 +1,16 @@
-name: 'CI - Frontend'
+name: "CI - Frontend"
```
Comment (phongcao) [inline .github/workflows/ci_frontend.yml]: Please use single quote to be consistent with other yaml files in our project.

File: .github/workflows/ci_frontend.yml [hunk 2]
```diff
@@ -83,11 +83,10 @@ jobs:
           message: ${{ env.COVERAGE }}
           color: green
           namedLogo: jest
-        
-      - if: steps.changes.outputs.src == 'true' && github.event_name != 'pull_request'
+      # We do not leave a PR Comment when merging since there is no PR
```
Comment (phongcao) [inline .github/workflows/ci_frontend.yml:87]: nit: add an empty line above.

File: .github/workflows/ci_frontend.yml [hunk 3]
```diff
@@ -72,7 +72,7 @@ jobs:
           # process.env.BRANCH = 'pull_27';
           echo "BRANCH=$(echo ${BRANCH_NAME})" >> $GITHUB_ENV
 
-      - if: steps.changes.outputs.src == 'true' && github.event_name != 'pull_request'
```
Comment (phongcao) [inline .github/workflows/ci_frontend.yml:75]: This condition was added from your previous PR to resolve the old issue. Wouldn't remove it now bring back that error?

**Review by phongcao — CHANGES_REQUESTED** [review]

File: .github/workflows/ci_frontend.yml (hunk 3 above)
Comment (mhassaninmsft) [reply .github/workflows/ci_frontend.yml:75]: As it turned out, this step is not what causes the failure, only the PR comment step below on line 86 is the one that causes it

**Review by argoldbe — APPROVED** [review]

**Review by phongcao — APPROVED** [review]

**Pull request merged**